site/content/docs/dashboards/dashboard-look-feel.md

```diff
@@ -8,12 +8,16 @@ menu:
 
 Dashboards have a lot of tweakable options to make them work for you in your situation.
 
-- **logo:** Display any image on the top left side of your dashboard.
-- **header:** Add a clarifying text like a team or environment name. This is separate from your custom URL or domain.
-- **tags:** Specify the tags of the checks you want to show on the dashboard. By default we show all checks. You can also choose to hide the tags from the dashboard.
-- **screen width:** Some screens are bigger than others, pick whether we should center the content or use the whole width.
-- **refresh rate:** Choose how often we should poll for new results and statuses.
-- **pagination:** Depending on your screen height and amount of checks, we can paginate through them so you have a full overview
-without sacrificing legibility per check.
+![config public monitoring dashboard 1](/docs/images/dashboards/look_feel1.png)
 
-![config public monitoring dashboard](/docs/images/dashboards/look_feel.png)
+- **title:** Add a title like "ACME status dash". This is separate from your custom URL or domain.
+- **description:** Add a clarifying text to explain visitors what they are looking at.
+- **checks:** Specify the tags of the checks you want to show on the dashboard. By default we show all checks.
+    You can also choose to hide the tags from the dashboard.
+
+![config public monitoring dashboard 2](/docs/images/dashboards/look_feel2.png)
+
+- **logo:** Display your company logo (or any image) on the top left side of your dashboard.
+- **logo link:** Direct visitors to your company website when they click the logo.
+- **favicon:** Put the cherry on top with a custom favicon.
+- **auto paginate:** Depending on your screen height and amount of checks, we can paginate through them so you have a full overview
```

Comment:
Small last thing, think there's a missing period at the end of the auto paginate description. After that, LGTM!